aya-build/src/lib.rs

```diff
@@ -22,7 +22,10 @@ use cargo_metadata::{Artifact, CompilerMessage, Message, Package, Target};
 /// prevent their use for the time being.
 ///
 /// [bindeps]: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html?highlight=feature#artifact-dependencies
-pub fn build_ebpf(packages: impl IntoIterator<Item = Package>) -> Result<()> {
+pub fn build_ebpf(
+    packages: impl IntoIterator<Item = Package>,
+    nightly_version: Option<String>,
```

Comment:
Why not &str?

Reply:
No particular reason, can change it to a reference.